website/databags/board.json

```diff
@@ -1,91 +1,94 @@
 {
   "types": [
     {
-      "name": "Current board members:",
+      "name": "Vorstand",
       "members": [
         {
-              "name": "Dr.-Ing. Mike Müller",
-              "role":"1. Vorsitzender",
-              "image": "/static/media/board/2020_mike.jpg",
-              "twitter":"",
-              "linkedin":"",
-              "xing":""
+          "name": "Dr.-Ing. Mike Müller",
+          "role": "1. Vorsitzender",
+          "image": "/static/media/board/2020_mike.jpg",
+          "twitter": "",
+          "linkedin": "",
+          "xing": ""
         },
         {
           "name": "Alexander Hendorf",
-          "role":"2. Vorsitzender",
-          "image": "/static/media/board/hendorf.png",
-          "twitter":"",
-          "linkedin":"",
-          "xing":""
+          "role": "2. Vorsitzender",
+          "image": "/static/media/board/hendorf.jpeg",
+          "twitter": "https://www.twitter.com/hendorf/",
+          "linkedin": "https://www.linkedin.com/in/hendorf/",
+          "xing": ""
         },
         {
           "name": "Dr. Tereza Iofciu",
-          "role":"1. stellvertretender Vorsitzender",
+          "role": "1. stellvertretender Vorsitzender",
           "image": "/static/media/board/tereza.png",
-          "twitter":"",
-          "linkedin":"",
-          "xing":""
+          "twitter": "",
+          "linkedin": "",
+          "xing": ""
         },
         {
           "name": "Prof. Dr. Martin Christen",
-          "role":"2. stellvertretender Vorsitzender",
+          "role": "2. stellvertretender Vorsitzender",
           "image": "/static/media/board/Martin_Christen.jpg",
-          "twitter":"",
-          "linkedin":"",
-          "xing":""
+          "twitter": "",
+          "linkedin": "",
+          "xing": ""
         },
         {
           "name": "Dr. Klaus Bremer",
-          "role":" Vorstandsmitglied",
+          "role": " Vorstandsmitglied",
           "image": "/static/media/board/2020_klaus.jpg",
-          "twitter":"",
-          "linkedin":"",
-          "xing":""
+          "twitter": "",
+          "linkedin": "",
+          "xing": ""
         },
         {
-          "name": "Katharine Jarmul ",
-          "role":" Beraterin",
-          "image": "/static/media/board/1517716762908.jpeg",
-          "twitter":"",
-          "linkedin":"",
-          "xing":""
+          "name": "Jessica Greene",
+          "role": " Vorstandsmitglied",
+          "image": "/static/media/board/jessica.jpeg",
+          "twitter": "",
+          "linkedin": "",
```

Comment:
do we want to add these social media links? I'm not on Xing and barely on Twitter (now X) but my LinkedIn is https://www.linkedin.com/in/jessica0greene/ maybe folks also have personal websites?

Reply:
We can add it but currently it does not show on the website. I think I can add links.